### PR TITLE
Dockerize + run on gcloud

### PR DESCRIPTION
**Pull request by max-mapper**

### CODE REVIEW

File: eDNAExplorer_Alpha.R [hunk 1]
```diff
@@ -133,6 +131,6 @@ function(First_Date,Last_Date,Marker,Num_Mismatch,TaxonomicRank,CountThreshold,F
       theme_bw()+geom_point()+geom_smooth()
   }
   #Save plot as json object
-  jfig <- plotly:::to_JSON(ggplotly(p))
+  jfig <- plotly_json(p, FALSE)
```
Comment (max-mapper) [inline eDNAExplorer_Alpha.R:134]: which one is correct?

File: eDNAExplorer_Alpha.R [hunk 2]
```diff
@@ -29,7 +27,7 @@ Sys.setenv("AWS_ACCESS_KEY_ID" = "<insert key>",
 #* @param EnvironmentalParameter:string Environmental variable to analyze against alpha diversity
 #* @param AlphaDiversity:string Alpha diversity metric
 #* @get /Tronko_Input
-function(First_Date,Last_Date,Marker,Num_Mismatch,TaxonomicRank,CountThreshold,FilterThreshold,EnvironmentalParameter,AlphaDiversity){
+alpha <- function(Object_Name,First_Date,Last_Date,Marker,Num_Mismatch,TaxonomicRank,CountThreshold,FilterThreshold,EnvironmentalParameter,AlphaDiversity){
```
Comment (max-mapper) [inline eDNAExplorer_Alpha.R:30]: @levisimons the function needs to be defined like this to be able to be imported from the api.R file

File: eDNAExplorer_Alpha.R (hunk 1 above)
Comment (levisimons) [reply eDNAExplorer_Alpha.R:134]: The one with the final line 'jfig <- plotly_json(p, FALSE)'

File: eDNAExplorer_Alpha.R [hunk 3]
```diff
@@ -15,10 +15,8 @@ require(htmlwidgets)
 require(plotly)
 require(jsonlite)
 
-Sys.setenv("AWS_ACCESS_KEY_ID" = "<insert key>",
```
Comment (max-mapper) [inline eDNAExplorer_Alpha.R:18]: @levisimons I removed these as I assume they will be set on the command line and I dont wanna hard code them in the script

File: eDNAExplorer_Alpha.R (hunk 1 above)
Comment (max-mapper) [reply eDNAExplorer_Alpha.R:134]: Ok so no changed needed to this PR

**Pull request merged**